### ISSUE TITLE
Fix CMS pagination markup

### ISSUE THREAD

**Issue by skoolbus39** [unlcms/project-herbie#972]
The list item that contains an ellipsis has a `[role=presentation]` attribute which fails the aXe Accessibility Metric.

> `<ul>` and `<ol>` must only directly contain `<li>`, `<script>` or `<template>` elements

and

> List element has direct children that are not allowed: `[role=presentation]`

HTML context:
```
<ol class="pager__items js-pager__items dcf-list-bare dcf-d-flex dcf-flex-wrap dcf-ai-center dcf-col-gap-2 dcf-row-gap-2">
```

**Comment by sarongebre** [comment]
I have recreated the issue here https://github.com/unlcms/unl_five/issues/86
